web-frontend/modules/core/search/types/base.js

```diff
@@ -34,4 +34,17 @@ export class BaseSearchType {
       descriptionSegments: [],
     }
   }
+
+  // Returns true if the result can be navigated to (has a valid URL)
```

Comment:
Super minor, but is there a reason why you're using inline comments (`// comment`) instead of JSDoc style like the rest of the codebase:

```
/**
 * Description
 */
method() {}
```